tests/test_playbooks/manifest_role.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: fetch status
+      include_tasks: tasks/status_info.yml
+    - name: ensure test organization
+      include_tasks: tasks/organization.yml
+      vars:
+        organization_name: "Test Organization"
+        organization_state: "present"
+
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  roles:
+    - role: manifest
+      vars:
+        manifest_download: false
```

Comment:
hmm, but you have download false... sure you don't have an "updated" manifest in https://github.com/theforeman/foreman-ansible-modules/tree/develop/tests/test_playbooks/data?

Reply:
oh, I think I know…